src/playwright/playwright.config.ts

```diff
@@ -9,8 +9,6 @@ import { type CoverageReporterOptions } from './coverage-reporter.js';
 export default defineConfig({
   expect: {
     toHaveScreenshot: {
-      maxDiffPixelRatio: 0.1,
```

Comment:
We should be able to remove this now that most of the dust has settled. As with the Webkit flakiness, I'll continue to keep an eye on it.